### DISCUSSION TITLE
Files Upload

### DISCUSSION THREAD

**Discussion by olaTechie** [strapi/strapi#6646] [Legacy Topics]
Good day,

Please, will like to confirm whether 'Files upload' plug-in is no longer available. I tried using quick start, and 'Files upload' plugin not available.

I will like to import csv files.

Thanks

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
